res/controllers/Traktor-Kontrol-S3-hid-scripts.js

```diff
@@ -235,7 +236,7 @@ TraktorS3.Deck.prototype.defineScaler = function(msg, name, deckOffset, deckBitm
 };
 
 TraktorS3.Deck.prototype.registerInputs = function(messageShort, messageLong) {
-    var deckFn = TraktorS3.Deck.prototype;
+    const deckFn = TraktorS3.Deck.prototype;
```

Comment:
The `const` keyword is not supported yet in 2.3. Please revert those jslint changes from 2.4.

Reply:
ah yes ok